### ISSUE TITLE
Userspace documentation: Page on how to use, set, etc App ID

### ISSUE THREAD

**Issue by ppannuto** [tock/book#56]
Capturing this here as a pointer to https://github.com/tock/libtock-c/pull/280, which is part of what should eventually be on a documentation page for this.

**Comment by bradjc** [comment]
We have https://book.tockos.org/course/policies#app-credentials and https://book.tockos.org/course/usb-security-key/key-hotp-access#step-1-credentialed-apps but I agree we should have something more of a users guide to what AppId is and how to use it. I think this has been blocked in some sense on getting support for a proper signature that really makes sense with the AppId infrastructure.

**Comment by ppannuto** [comment]
Yeah, I think things are that are within courses aren't really obvious as standalone documentation. I don't think people would expect to look inside of a tutorial/course for a reference
